src/JustEat.StatsD/JustEat.StatsD.csproj

```diff
@@ -5,14 +5,19 @@
     <PackageId>JustEat.StatsD</PackageId>
     <RootNamespace>JustEat.StatsD</RootNamespace>
     <Summary>A .NET library for publishing metrics to statsd.</Summary>
-    <TargetFrameworks>net451;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net451;netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
I thought we were going to avoid adding extra TFMs?

Reply:
To fully take advantage of `Span<T>` we'll want to use `netcoreapp2.1`, or wait until `netstandard2.1` (which `netcoreapp2.1` won't implement btw! only `netcoreapp3.0` and up). 

If we wait until `netstandard2.1` we will want to multi-target to keep support for current clients, so we may as well embrace `netcoreapp2.1` today is my thought.

Reply:
I meant more non-standard TFMs (i.e. the app ones), rather than any TFMs at all.

Is there some guidance somewhere on advantages vs. disadvantages to targeting the more specific implementations?

Reply:
There will be guidance from Microsoft soon apparently, still waiting.

There are pros and cons, the bottom line is if you want to use the new APIs today, you have to target the platforms directly to use them. netstandard will always trail behind. For this sort of library, it feels appropriate to add this target.

Reply:
Looks like the guidance is “_consider_”: https://twitter.com/ben_a_adams/status/1039989566021820418?s=21